03 - Data Science and Custom Skills/Azure Function Skills/SampleSkills/0_CustomSkillTemplate.cs

```diff
@@ -0,0 +1,171 @@
+/****************************************************************************************
+ * 
+ * This template should make it easy for you to create a custom skill as
+ * it takes care of most of the boilerplate code for serialization and  deserialization.
+ * 
+ * STEPS:
+ * 1. Define the input fields for your skill.
+ *    Modify the InputRecordData class.
+ * 
+ * 2. Define the output fields for your skill.
+ *    Modify the OutputRecordData class.
+ *    
+ * 3. Define what action your will will take to enrich/transform the input into the output.
+ *    Modify the DoWorkHere class.
```

Comment:
There is no DoWorkHere class, just the DoWork method which should be modified

Reply:
Fixed my comment.